app/bundles/LeadBundle/Form/Type/CampaignConditionLeadPageHitType.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Mautic\LeadBundle\Form\Type;
+
+use Mautic\CoreBundle\Form\DataTransformer\SecondsConversionTransformer;
+use Mautic\PageBundle\Form\Type\PageListType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\Form\FormInterface;
+
+/**
+ * @codeCoverageIgnore
+ */
```

Comment:
Please do not use these. It skews the code coverage report. Instead, you can create a functional test that loads the edit page of this form like the campaign builder does. That's way more helpful.
```suggestion
```